codeSnippets/snippets/tutorial-website-static/src/main/kotlin/com/example/plugins/Routing.kt

```diff
@@ -6,8 +6,6 @@ import io.ktor.server.routing.*
 
 fun Application.configureRouting() {
     routing {
-        static("/static") {
-            resources("files")
-        }
+        staticResources("/static", "files")
```

Comment:
It's better to change only one thing in the pull request (JVM suffixes or static resources).

Reply:
Yes, you are right. I will keep it in mind for future requests!